### ISSUE TITLE
Form field missing a label (transcript drop-down box)

### ISSUE THREAD

**Issue by elynema** [samvera-labs/ramp#225]
Sites identified: MCO
Page(s) identified: Media object page, transcript display

WCAG compliance level: A

Description:

Every form element should have a descriptive text label.

Unclear labels make form-filling harder for everyone.

A label may already be visible on the page — but to be accessible, that label needs to be associated with the form element in the HTML. This helps visitors using screen readers to understand what information is required, while also making it easier for speech-input users to control the form element.

More details at [SiteImprove example](https://my2.siteimprove.com/Inspector/1167098/A11Y/Page?pageId=40886721775&impmd=DB21396D67119C3E30B314CCF5B34C39&conf=a+aa+aria+si&wcag=twopointone&decision=dismissed+falsePositive+none#/sia-r8/failed/MkhUVnFjOHg=;ZjcxMDQ1MTY=).

[Private Zenhub Image](https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBblpnIiwiZXhwIjpudWxsLCJwdXIiOiJibG9iX2lkIn19--f7cc25a16d52eb0c9109d27201119044e62de97f/image.png)

**Comment by elynema** [comment]
I created this an issue in the Ramp repository, as I believe this is a problem to be resolved within Ramp.

**Comment by joncameron** [comment]
Resolved in Ramp as part of a previous refactor.

**Comment by elynema** [comment]
Checked the new media object page on avalon-dev with the SiteImprove browser plugin and this error no longer pops up.
